lib/pat.c

```diff
@@ -1780,6 +1780,24 @@ delinfo_new(grn_ctx *ctx,
 
 /* pat operation */
 
+static inline uint32_t
+pat_node_get_w_of_pat_entry(uint32_t flags)
```

Comment:
* We should not use the `pat_node_` prefix here because this is not a method of `pat_node`
* We should not use the `get` here because this doesn't get something

Reply:
I changed to `compute_bit_width_of_pat_entry` from `pat_node_get_w_of_pat_entry`.